packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -643,8 +646,9 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
         <IconButton
           className={'ms-ComboBox-CaretDown-button'}
           styles={this._getCaretButtonStyles()}
-          role="presentation"
+          role={isButtonAriaHidden ? 'presentation' : undefined}
           aria-hidden={isButtonAriaHidden}
+          {...(isButtonAriaHidden ? {} : labelProps)}
```

Comment:
I believe this should work and it will avoid (potentially) creating an object every render.

```suggestion
          {...(!isButtonAriaHidden ? labelProps : undefined)}
```